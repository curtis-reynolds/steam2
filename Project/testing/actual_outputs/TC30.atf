00_sellstd_________SS_000000.00
