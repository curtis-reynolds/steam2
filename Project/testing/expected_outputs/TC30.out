Login successful.
A game with the same name already exists. Please choose a unique name.
